cmd/kube-apiserver/app/server.go

```diff
@@ -178,6 +181,9 @@ func (s *APIServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.BasicAuthFile, "basic-auth-file", s.BasicAuthFile, "If set, the file that will be used to admit requests to the secure port of the API server via http basic authentication.")
 	fs.StringVar(&s.ClientCAFile, "client-ca-file", s.ClientCAFile, "If set, any request presenting a client certificate signed by one of the authorities in the client-ca-file is authenticated with an identity corresponding to the CommonName of the client certificate.")
 	fs.StringVar(&s.TokenAuthFile, "token-auth-file", s.TokenAuthFile, "If set, the file that will be used to secure the secure port of the API server via token authentication.")
+	fs.StringVar(&s.OpenIDEndpoint, "openid-endpoint", s.OpenIDEndpoint, "If set, the endpoint will be used to verify the JWT using OpenID protocol")
+	fs.StringVar(&s.OpenIDClientID, "openid-client-id", s.OpenIDClientID, "The client ID for the OpenID client, must be set if openid-endpoint is set")
+	fs.StringVar(&s.OpenIDServerCertFile, "openid-server-cert", s.OpenIDServerCertFile, "The PERM-encoded x509 certificate for the OpenID server, optional")
```

Comment:
`PEM-encoded`, `OpenIDCAFile`, `openid-ca-file` (to follow client-ca-file pattern above)